.github/workflows/tests.yml

```diff
@@ -86,6 +86,7 @@ jobs:
       fail-fast: false
       matrix:
         bazel_version: [6.5.0, 7.1.0]
+        sandbox: [true, false]
```

Comment:
Can you add this info to the `name` of the step as well so its possible to tell the difference in github

Reply:
right

Reply:
updated, probably need to update the required checks as well so this can be landed

Reply:
i can do that now